automl/cloud-client/get_dataset_test.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import get_dataset
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
```

Comment:
nit: Strong preference to a wrapper to throw an error if this isn't set (like we have in Java)

Reply:
Agreed, though I believe the error occurs even further up.
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/conftest.py#L20

Reply:
Python will throw an error if it isn't set.